SimPEG/Examples/Inversion_IRLS.py

```diff
@@ -0,0 +1,132 @@
+from SimPEG import *
+
+
+def run(N=200, plotIt=True):
+    """
+        Inversion: Linear Problem
+        =========================
+
+        Here we go over the basics of creating a linear problem and inversion.
```

Comment:
A bit more explanation on what this script is doing and why one might use IRLS would be helpful!